[2025-02-03T21:28:56.166-0300] {processor.py:186} INFO - Started process (PID=299858) to work on /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-03T21:28:56.173-0300] {processor.py:914} INFO - Processing file /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-03T21:28:56.203-0300] {logging_mixin.py:190} INFO - [2025-02-03T21:28:56.194-0300] {dagbag.py:588} INFO - Filling up the DagBag from /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-03T21:28:56.254-0300] {logging_mixin.py:190} INFO - [2025-02-03T21:28:56.244-0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-03T21:28:56.256-0300] {logging_mixin.py:190} INFO - [2025-02-03T21:28:56.255-0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-03T21:28:56.260-0300] {processor.py:927} WARNING - No viable dags retrieved from /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-03T21:28:56.357-0300] {processor.py:208} INFO - Processing /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.227 seconds
[2025-02-03T22:19:10.038-0300] {processor.py:186} INFO - Started process (PID=43857) to work on /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-03T22:19:10.071-0300] {processor.py:914} INFO - Processing file /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-03T22:19:10.081-0300] {logging_mixin.py:190} INFO - [2025-02-03T22:19:10.079-0300] {dagbag.py:588} INFO - Filling up the DagBag from /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-03T22:19:10.094-0300] {logging_mixin.py:190} INFO - [2025-02-03T22:19:10.092-0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-03T22:19:10.096-0300] {logging_mixin.py:190} INFO - [2025-02-03T22:19:10.095-0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-03T22:19:10.097-0300] {processor.py:927} WARNING - No viable dags retrieved from /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-03T22:19:10.135-0300] {processor.py:208} INFO - Processing /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.125 seconds
[2025-02-03T22:21:30.383-0300] {processor.py:186} INFO - Started process (PID=47002) to work on /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-03T22:21:30.731-0300] {processor.py:914} INFO - Processing file /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-03T22:21:30.743-0300] {logging_mixin.py:190} INFO - [2025-02-03T22:21:30.742-0300] {dagbag.py:588} INFO - Filling up the DagBag from /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-03T22:21:30.758-0300] {logging_mixin.py:190} INFO - [2025-02-03T22:21:30.756-0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-03T22:21:30.760-0300] {logging_mixin.py:190} INFO - [2025-02-03T22:21:30.759-0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-03T22:21:30.761-0300] {processor.py:927} WARNING - No viable dags retrieved from /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-03T22:21:30.804-0300] {processor.py:208} INFO - Processing /home/gnobisp/Documents/floder1/code-challenge/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.448 seconds
